resources/providers/azurelib/governance/management_group.go

```diff
@@ -105,9 +112,10 @@ func (p *provider) scan(ctx context.Context) (map[string]Subscription, error) {
 
 		mg := managementGroups[strings.FromMap(parent, "name")]
 		subscriptions[asset.SubscriptionId] = Subscription{
```

Comment:
Question: what will be the format of the **key** of the returned map `GetSubscriptions(...) (map[string]Subscription, error)`?

Because this key is used in `resources/fetching/fetchers/azure/insights_batch_fetcher.go` and is copied to `AzureAsset.SubscriptionId` while transforming azure SDK models to `AzureAsset`

Reply:
Key map === asset.SubscriptionId === Subscription.ShortID

Reply:
Key map === asset.SubscriptionId === Subscription.ShortID

Reply:
[Not thoroughly thought idea] Perhaps we can create two string aliases like `type SubscriptionIDFull string`  and `type SubscriptionIDShort`, which might help in each case to know what kind of ID we have (even as map keys or everywhere else).